qiskit/circuit/controlledgate.py

```diff
@@ -29,7 +29,6 @@ def __init__(self, name, num_qubits, params, label=None, num_ctrl_qubits=1,
 
         Attributes:
             num_ctrl_qubits (int): The number of control qubits.
-            ctrl_state (int): The control state in decimal notation.
```

Comment:
`ctrl_state` is an instance attribute. Was it being marked as not (because it shares the name of a kwarg)?

Reply:
I'd have to look at the specific warning from the past runs (I don't remember this particular one, there were >400 of them). But if I removed it then yes it was probably saying it couldn't find the attribute ctrl_state.